core/templates/components/state-directives/answer-group-editor/summary-list-header.component.ts

```diff
@@ -29,12 +34,17 @@ export class SummaryListHeaderComponent {
  @Input() summary: string;
  @Input() shortSummary: string;
  @Input() isActive: boolean;
- @Input() onDeleteFn: (index: number, event: Event) => void ;
+ @Output('onDeleteFn') deleteEventEmitter: EventEmitter<EventData> =
+  new EventEmitter();
```

Comment:
A rule we follow, Never prefix @Outputs with `on`. It can cause clashes with the existing DOM apis.
```suggestion
 @Output() summaryDelete: EventEmitter<EventData> =
  new EventEmitter();
```

Reply:
Done.